src/bundle/Resources/views/layout.html.twig

```diff
@@ -162,6 +162,7 @@
             '@EzPlatformAdminUiBundle/Resources/public/js/scripts/helpers/request.helper.js'
             '@EzPlatformAdminUiBundle/Resources/public/js/scripts/helpers/notification.helper.js'
             '@EzPlatformAdminUiBundle/Resources/public/js/scripts/helpers/timezone.helper.js'
+            '@EzPlatformAdminUiBundle/Resources/public/js/scripts/helpers/content.type.icon.helper.js'
```

Comment:
I would name this just `content.type.helper.js` because in the `addConfig()` the group for those methods is `helpers.contentType`, which implies there will be different methods for content types not only of the icon.

Reply:
I was thinking about the same and I agree.